app/services/irs_attempts_api/attempt_event.rb

```diff
@@ -34,7 +34,7 @@ def self.from_jwe(jwe, private_key)
       event_type = parsed_event['events'].keys.first.split('/').last
       event_data = parsed_event['events'].values.first
       AttemptEvent.new(
-        jti: parsed_event['jti'],
+        jti: parsed_event['jti'].split(':').last,
```

Comment:
do we want to extract out the `key_id` as a property on the event now too?

Reply:
I swear I implemented that after this comment, but apparently it's not here. Coming soon to a PR near you!

Reply:
This story is dragging on (my fault), but having just implemented this, I don't like it and think I want to change the model a bit.

tl;dr, it doesn't make sense to me to set key id but not be able to set the key itself. It's logical enough in `from_jwe` that we'd capture that information, but if you were to then call `key_id` you'd get a key id that might not match what `event_data_encryption_key` returns.

Right now it's implemented in the yaml like so:

```yml
  irs_attempt_api_public_key: change-me-pls
  irs_attempt_api_public_key_id: key1
```

This seems easy enough for currently only supporting one key. If we're going to keep this, I think we should not extract the `key_id` parameter as proposed in this comment.

But I feel like what we actually want is something like:

```yml
  irs_attempt_api_public_keys:
    key1: change-me-pls
```

Then, setting `key_id` in code would cause the correct public key to be looked up.

What do you think, @zachmargolis et al.? Should I go ahead with that?

Reply:
honestly, I'm not sure I follow anymore so I trust your call

Reply:
OK. I'm going to propose leaving this as-is for now and not explicitly capturing `key_id` as an attribute inside AttemptEvent.

What I proposed with application.yml.default changes turns out to not be quite how IdentityConfig works, and feels a little too close to implementing actual multi-key support. (Jack also pointed out that it would probably make more sense to implement storage of that in the database when the time comes.)